### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by KhanjanDabhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,36 @@
...
+I am a recent graduate from Lakehead University with a degree in Software Engineering. Organized and dependable candidate successful at managing multiple priorities with a positive attitude. Willingness to take on added responsibilities to meet team goals. During my undergrad, I have studied performance analysis, compilers, algorithm design, operating systems, applied computational intelligence, parallel programming, machine learning and much more. I have worked on projects ranging from database management, scripting and refactoring Java programs to data modelling, data visualization and developing neural networks in Python. I am currently working at Jarvis to improve my skill sets as a developer while building challenging projects and applications using new technologies such as Docker, Springboot, Maven, REST API, RDBMS, etc. I strive to become a better developer through process of trial and error. Consistency, taking responsibility for your own work/actions and being diligent are traits that help me become a better verson of myself everyday.
```
Comment (jarviscanada) [inline README.md]: I strive to become a better developer through a process of trial and error

Comment (jarviscanada) [reply README.md]: Consistency, taking responsibility for your own work/actions and being diligent are traits that help me become a better version of myself every day.

File: README.md [hunk 2]
```diff
@@ -1,52 +1,36 @@
...
-
-**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/hadoop)]: In Progress
-
-**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/spark)]: In Progress
-
-**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/cloud_devops)]: In Progress
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KhanjanDabhi/tree/master/linux_sql)]: Developed a Linux Cluster Monitoring agent that allows a user to keep track of machine specs and usage data across numerous Linux systems. The agent collects machine hardware information using Bash scripts, which is subsequently saved in a Docker-provisioned PostgreSQL database. Crontab is used to collect resource utilisation data on a regular basis, which may then be evaluated using SQL queries. All of the testing was done on a Google Cloud Platform virtual machine running CentOS7 distribution of Linux. Every script was manually checked to ensure that it met all of the requirements.
```
Comment (jarviscanada) [inline README.md]: Developed a Linux Cluster Monitoring agent that allows users to keep track of machine specs and usage data across numerous Linux systems. The agent collects machine hardware information using Bash scripts, subsequently saved in a Docker-provisioned PostgreSQL database. Crontab is used to collect resource utilization data automatically, which may then be analyzed using SQL queries. All of the testings were done on a Google Cloud Platform virtual machine running CentOS7 distribution of Linux. Every script was manually checked to ensure that it met all of the requirements.

File: README.md [hunk 3]
```diff
@@ -1,52 +1,36 @@
...
 
-**Refactoring and Object-Oriented Design** [[GitHub](https://github.com/KhanjanDabhi/Refactoring)]: On given Movie Rental System, performed refactoring step-by-step to make it easier to add new code, fixed smelly code to make sure that the concepts of object-orientation are preserved/applied, recover parts of the system design by generating class diagrams of the system, made sure that all the unit test passed after applying changes.
+**Refactoring and Object-Oriented Design** [[GitHub](https://github.com/KhanjanDabhi/Refactoring)]: Performed refactoring on a given Movie Rental System which was designed in Java to operate on a command-line interface. Performed refactoring step-by-step to make it easier to add new code, fixed smelly code to make sure that the concepts of object-orientation are preserved/applied, recover parts of the system design by generating class diagrams of the system, making sure that all the unit tests passed after applying changes. Developed a strategy where refactoring is carried out in small steps, with the tests being run after each change so that one can quickly recover from any faults that are introduced as part of the refactoring process.
```
Comment (jarviscanada) [inline README.md]: Performed refactoring on a given Movie Rental System programmed in Java to operate on a command-line interface. Performed refactoring step-by-step to make it easier to add new code, fixed smelly code to make sure that the concepts of object-orientation are preserved/applied, recover parts of the system design by generating class diagrams of the system, making sure that all the unit tests passed after applying changes. Developed a strategy where refactoring is carried out in small steps, with the tests being run after each change to quickly recover from any faults introduced as part of the refactoring process.

File: README.md [hunk 4]
```diff
@@ -1,52 +1,36 @@
...
+**Applied Computational Intellegence** [[GitHub](https://github.com/KhanjanDabhi/Applied-Computational-Intelligence)]: This project was an analysis of various pre-trained and self-trained neural networks such as VGG16, ResNet50 and Mobile-Net. These models were compared for their classification accuracy in Python using Keras and TensorFlow. They were trained on the same dataset which contained images for CT scans of lungs with and without an infection. Using Pandas, Numpy and Scipy, key concepts of machine learning for images were applied on the dataset such as data augmentation, adaptive histogram equalization, contrast-stretching and cross-validation. SKlearn was used to perform hyperparameter tuning on each of the above-mentioned models and changes in accuracy w.r.t. tuning were manually recorded.  Effects of image segmentation on these models were also noted by performing un-supervised segmentation through K-means clustering. Quantitative analysis was done at the end from the collected data to generate a properly formatted report in LaTeX.
```
Comment (jarviscanada) [inline README.md]: **Applied Computational Intelligence** [[GitHub](https://github.com/KhanjanDabhi/Applied-Computational-Intelligence)]: This project was an analysis of various pre-trained and self-trained neural networks such as VGG16, ResNet50 and Mobile-Net. These models were compared for their classification accuracy in Python using Keras and TensorFlow. They were trained on the same dataset, which contained CT scans of lungs with and without an infection. Used Pandas, Numpy and Scipy, key concepts of machine learning for images were applied on the dataset such as data augmentation, adaptive histogram equalization, contrast-stretching and cross-validation. SKlearn was used to perform hyperparameter tuning on each of the models mentioned above, and changes in accuracy w.r.t. tuning were manually recorded. Effects of image segmentation on these models were also noted by performing un-supervised segmentation through K-means clustering. Quantitative analysis was done at the end from the collected data to generate a properly formatted report in LaTeX.

File: README.md [hunk 5]
```diff
@@ -1,52 +1,36 @@
...
 
 
 ## Professional Experiences
 
-**Software Developer, Jarvis (2021-present)**:  Worked on Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+**Software Developer, Jarvis (2021-present)**:  I've worked several of projects that required the use of Linux, Bash, Docker, Postgres, Java, and Spring Boot. In every project, I employed the scrum agile methodology with Git and Git Flow. As a scrum leader, I assisted people with project-related challenges and led daily scrum meetings.
```
Comment (jarviscanada) [inline README.md]: **Software Developer, Jarvis (2021-present)**: Working on several projects that required the use of Linux, Bash, Docker, Postgres, Java, and Spring Boot. I employed the Scrum/Agile methodology with Git and Git Flow in every project. As a scrum leader, I assisted people with project-related challenges and led daily scrum meetings.

**Review by jarviscanada — APPROVED** [review]
